Doc-804: Document new skin settings (#1931)
tinymce/tinymce-docs: tinydrive/tinydrive-api/standalone.md

});
```

### `skin`

> This option allows you to specify the skin that {{site.cloudfilemanager}} should use. The default skin included with {{site.cloudfilemanager}} is named "oxide".

Reviewer: ```suggestion
This option sets the skin applied to {{site.cloudfilemanager}}. The default skin included with {{site.cloudfilemanager}} is named "oxide".
```

**Type:** `String`

> **Default Value:** `oxide`
> 
> **Possible Values:** `oxide`, `oxide-dark`

Reviewer 2: ```suggestion
**Default Value:** `'oxide'`

**Possible Values:** `'oxide'`, `'oxide-dark'`
```
